WordPress/twentytwentyfour
# Faq Pattern Section

Faq section designed

**Description**

faq section designed as pattern

Reference to the issue that the PR fixes:
https://github.com/WordPress/twentytwentyfour/issues/119

**Screenshots**
https://prnt.sc/CDb7GJTljavD

**Testing Instructions**

1.  Activate the theme.
2.  Create a pettern named faq
3.  Add the code viewing as code editor.
4.  See the changes

**Another way**

1.     copy the code
2.     Create a faq php file and see the changes
